Oms indicators docs (#124)
msupply-foundation/msupply_docs: content/docs/programs/program_indicators.md

+++
title = "Program Indicators"
description = "Program Indicators"
date = 2023-05-19T00:00:00+00:00
updated = 2023-05-19T00:00:00+00:00
draft = false
weight = 62
sort_by = "weight"
template = "docs/page.html"

[extra]
lead = "Collecting data metrics across mSupply stores"
toc = true
top = false
+++

Program Indicators are essentially sets of configurable survey questions that can be answered as part of a program requisition. They are recorded against the period that the program requisition is created for and used in reporting.

## Enabling Program Indicators

Program Indicators are configured on the mSupply Primary Server, following the documenation [here](https://docs.msupply.org.nz/items:programs#adding_indicators_to_a_program).

## Manual Customer Requisitions

Manual Customer Requisitions can be made to other stores as a fallback if device that store is meant to be active on is no longer operational.

> 1. Create a manual requisition to another store as described [here](http://127.0.0.1:1111/docs/distribution/requisitions/#manual-requisition)

Reviewer: ```suggestion
1. Create a manual requisition to another store as described [here](/docs/distribution/requisitions/#manual-requisition)
```

So it can be used anywhere :P

Author: not even sure how that snuck in there!

2. Click on the Indicators tab

![Indicators tab](/docs/programs/images/indicators.png)

> Note: HIV is only available on Programs where "HIV" has been enabled.

Reviewer: ```suggestion
<div class='note'>
HIV is only available on Programs where "HIV" has been enabled.
</div>```

description = "Program Indicators"
date = 2023-05-19T00:00:00+00:00
updated = 2023-05-19T00:00:00+00:00
draft = false
weight = 62
sort_by = "weight"
template = "docs/page.html"

[extra]
lead = "Collecting data metrics across mSupply stores"
toc = true
top = false
+++

Program Indicators are essentially sets of configurable survey questions that can be answered as part of a program requisition. They are recorded against the period that the program requisition is created for and used in reporting.

## Enabling Program Indicators

Program Indicators are configured on the mSupply Primary Server, following the documenation [here](https://docs.msupply.org.nz/items:programs#adding_indicators_to_a_program).

## Manual Customer Requisitions

Manual Customer Requisitions can be made to other stores as a fallback if device that store is meant to be active on is no longer operational.

1. Create a manual requisition to another store as described [here](/docs/distribution/requisitions/#manual-requisition)
2. Click on the Indicators tab

![Indicators tab](/docs/programs/images/indicators.png)

<div class='note'>
HIV is only available on Programs where "HIV" has been enabled.

> </div>```

Reviewer: ```suggestion
</div>
```

Author: lol thanks github
